src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -26,4 +34,44 @@ TableToolbar.propTypes = {
    * Function used to translate parent table element in fullscreen
    */
   toggleFullscreen: PropTypes.func.isRequired,
+  /*
+   * Boolean value define if data is fully loaded
+   */
+  isReady: PropTypes.bool.isRequired,
+  /*
+   * Boolean value which defined if user can actually do modifications on the webapp
+   */
+  editMode: PropTypes.bool.isRequired,
+  /*
+   * Function used to import a file in Table. If undefined, import button is not displayed
+   */
+  onImport: PropTypes.func,
+  /*
+   * Function used to export a file in Table. If undefined, export button is not displayed
+   */
+  onExport: PropTypes.func,
+  /*
+   * List of extra React elements to add in the TableToolbar
+   */
+  customToolbarActions: PropTypes.array,
+  /**
+   * Component's labels:
+   * Structure:
+   * <pre>
+     {
+      label: 'string'
+    }
+   </pre>
+   */
+  labels: PropTypes.shape({
+    clearErrors: PropTypes.string,
+    label: PropTypes.string,
+    loading: PropTypes.string,
+    errorsPanelMainError: PropTypes.string,
+    placeholderTitle: PropTypes.string,
+    placeholderBody: PropTypes.string,
+    import: PropTypes.string,
+    export: PropTypes.string,
+    fullscreen: PropTypes.string,
+  }),
```

Comment:
Only 3 labels are actually used in this component (fullscreen, import and export), you can remove the other keys.
Also, could you update the "Structure" in the comment above with the actual keys for these labels please? (example [here](https://github.com/Cosmo-Tech/webapp-component-ui/blob/main/src/menus/HelpMenu/HelpMenu.js#L122-L134))

Reply:
DONE